src/client/webpack.config.js

```diff
@@ -43,7 +42,12 @@ const webpackConfig = {
     ]),
     new webpack.optimize.CommonsChunkPlugin(/* chunkName= */'vendor', /* filename= */'vendor.js', function(module){
       return module.resource && module.resource.indexOf('node_modules') !== -1;
-    })
+    }),
+    new webpack.DefinePlugin({
+     __CONFIG__: args.endpoint
```

Comment:
Prod deployment is broken at the moment in dev because the build script runs `npm run deploy:prod` which breaks trying to find the `config.json` import. If we keep the way things are then I'll add a config file for prod which doesn't override the endpoints. It'll effectively be an empty `config.json` which gets imported into bootstrapper. Happy to go that way?